install/mysql-install.sh

```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
   mc
 msg_ok "Installed Dependencies"
 
+PS3="Please select a Major Release: "
+RELEASES=("8.0" "8.4 - LTS" "9.1 - Innovation")
+select RELEASE in "${RELEASES[@]}"; do
+  case ${RELEASE} in
+    "8.0")
```

Comment:
Any particular reason to include this in addition to `8.4 - LTS`

Reply:
some scripts work only on MySQL 8.0.xx 🥴😅